WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -264,6 +264,8 @@ import AutomatticTracks
     /// This is set to true after the Reader Manage view is dismissed
     var shouldForceRefresh = false
 
+    lazy var isSidebarModeEnabled = splitViewController?.isCollapsed == false
```

Comment:
I don't think this property needs to be stored? It should be okay to just query within the table view delegate implementation right? Storing this property would cause an issue when switching between full-screen and split-screen mode.

Reply:
I plan to rework it in the upcoming Reader project and make some other tweaks to the feed. The cells currently don't support dynamically switching from "regular" to "compact" mode. The `.compact` view controller creates new instances of reader screens, so this property is re-created.